src/components/Modals/ConfirmTransaction/index.tsx

```diff
@@ -64,151 +66,157 @@ const ConfirmTransaction: React.FC = () => {
+                      </StyledLink>
+                    </Flex>
+                  </StyledWarningContainer>
+                </StyledGraphAndWarningContainer>
+                <Spacer size={'sm'} />
               </div>
-            </Flex>
-            <Spacer size={'xs'} />
-            <StyledDetailHeader>
-              <P2 color={colors.white}>Deposit</P2>
-            </StyledDetailHeader>
-            <StyledContentContainer>
-              <P2 color={'#000'}>{inputValue} wPOKT</P2>
-            </StyledContentContainer>
-            <Spacer size={'xs'} />
-            <StyledDetailHeader>
-              <P2 color={colors.white}>Address</P2>
-            </StyledDetailHeader>
-            <StyledContentContainer copied={isCopied}>
-              <Flex align={'center'} justify={'space-between'}>
-                <P2 id={'address'} color={'#000'}>
-                  {address ? shortenAddress(address, 10) : ''}
-                </P2>
-                <button onClick={onCopy}>{!isCopied ? <CopySvg /> : 'Copied!'}</button>
-              </Flex>
-            </StyledContentContainer>
-          </div>
-        ) : (
-          <div>
-            <StyledDetailHeader>
-              <P2 color={colors.white}>Amount to withdraw</P2>
-            </StyledDetailHeader>
-            <StyledContentContainer>
-              <P2 color={'#000'}>{inputValue} wPOKT</P2>
-            </StyledContentContainer>
-            <Spacer size={'xs'} />
-            <StyledGraphAndWarningContainer>
-              <StyledGraphContainer>
-                <RewardsGraphSvg />
-              </StyledGraphContainer>
-              <StyledWarningContainer>
-                <StyledWarning>
-                  <P3 color={colors.white}>Give it a second thought...</P3>
-                  <Spacer size={'xs'} />
-                  <P3 color={colors.white}>If you keep your stake longer you could earn more rewards.</P3>
-                </StyledWarning>
-                <Flex
-                  css={`
-                    width: 100%;
-                  `}
-                  justify={'space-between'}
-                >
-                  <StyledLink>
-                    <P2 color={colors.white}>FAQ</P2>
-                    <MultiplierSvg />
-                  </StyledLink>
-                  <StyledLink>
-                    <P2 color={colors.white}>Stats</P2>
-                    <MultiplierSvg />
-                  </StyledLink>
-                </Flex>
-              </StyledWarningContainer>
-            </StyledGraphAndWarningContainer>
-            <Spacer size={'sm'} />
+            )}
+            <StyledDepositButtonContainer>
+              {actionType === 'deposit' ? (
+                <button onClick={onDeposit}>
+                  <DepositButtonActiveSvg />
+                </button>
+              ) : (
+                <button onClick={onWithdraw}>
+                  <WithdrawButtonActiveSvg />
+                </button>
```

Comment:
I just noticed... I think we have a bunch of buttons that have SVGs and nothing else in them—this isn't very accessible. We should have at least some visually-hidden text to do this. The [Visually Hidden](https://reach.tech/visually-hidden) component from Reach UI can accomplish this. We should do this for all buttons that have this exact situation. We can probably do this in another PR though 😄

Reply:
Issue moved to #18